Makefile

```diff
@@ -128,15 +128,30 @@ tidy:
 	GO111MODULE=on go mod tidy
 	git diff --quiet go.mod go.sum
 
-image:
+image: image-chaos-daemon image-chaos-mesh image-chaos-fs image-chaos-scripts image-chaos-grafana image-chaos-dashboard image-chaos-kernel
```

Comment:
I made this change because building chaos-kernel image is too slow (on my minikube virtual machine).

Reply:
Do we need to change CI scripts?

Reply:
No. I only split `image` into multiple subtasks, so there is no need to change CI scripts.